### ISSUE TITLE
K8s in Non-Java IDE: java.lang.NoClassDefFoundError: com/intellij/openapi/projectRoots/JavaSdk

### ISSUE THREAD

**Issue by ivanporty** [GoogleCloudPlatform/cloud-code-intellij#2736]
```
java.lang.NoClassDefFoundError: com/intellij/openapi/projectRoots/JavaSdk
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldExecutorService.getJavaSdkEnvironment(SkaffoldExecutorService.kt:425)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldExecutorService.executeSkaffold(SkaffoldExecutorService.kt:168)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldExecutorService.executeSkaffold$default(SkaffoldExecutorService.kt:119)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldExecutorService.getSkaffoldOutputBlocking(SkaffoldExecutorService.kt:402)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.init.SkaffoldInitService.canGenerateConfiguration(SkaffoldInitService.kt:97)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.init.SkaffoldInitService$notifyUserIfGenerationPossible$1$canGenerateConfiguration$1.invokeSuspend(SkaffoldInitService.kt:201)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:740)
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.projectRoots.JavaSdk PluginClassLoader[com.google.gct.core, 20.4.1] com.intellij.ide.plugins.cl.PluginClassLoader@283da084
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:115)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 11 more
```

app.name.version
PyCharm
app.version.major
2020
app.version.minor
1
java.version
11.0.6
java.vm.vendor
JetBrains s.r.o
os.name
Linux
plugin.version
20.4.1

**Comment by ivanporty** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-intellij-internal](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/1737)

**Comment by ivanporty** [comment]
Confirmed reports from PyCharm and RubyMine

**Comment by ivanporty** [comment]
Fixed: https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/1737

**Comment by etanshaul** [comment]
If anyone is still encountering this, this was patched in Cloud Code release `20.4.2`.
